Dockerfile

```diff
@@ -31,23 +33,24 @@ RUN set -xe; \
     mix deps.compile --all; \
     mix release
 
-FROM alpine:3.16.2 as release
+FROM debian:11.6-slim as release
 
 RUN set -xe; \
-    apk add --update  --no-cache --virtual .runtime-deps \
+        apt-get update && \
+        TZ=Etc/UTC \
+        apt-get install -y \
         ca-certificates \
-        libmcrypt \
         libmcrypt-dev \
         openssl \
-        libstdc++ \
-        ncurses-libs \
+        libssl-dev\
```

Comment:
Since this is the release container, `libmcrypt-dev` can probably be replaced with `libmcrypt4` and `libssl-dev` shouldn't be necessary.